tests/unit/plugins/module_utils/test_opennebula.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2023, Michal Opala <mopala@opennebula.io>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import os
+import pytest
+import textwrap
+
+from ansible_collections.community.general.plugins.module_utils.opennebula import extend, flatten, render
```

Comment:
Just a bit of nit picking:
```suggestion
import os
import textwrap

import pytest

from ansible_collections.community.general.plugins.module_utils.opennebula import extend, flatten, render
```
So we have separate "blocks" for: builtin modules, 3rd party modules, modules in this codebase, as I believe is the good practice in pythonverse.